generators/entity/index.js

```diff
@@ -877,8 +879,11 @@ class EntityGenerator extends BaseBlueprintGenerator {
                 context.relationships.forEach(relationship => {
                     const otherEntityName = relationship.otherEntityName;
                     const otherEntityData = this.getEntityJson(otherEntityName);
-                    if (otherEntityData && otherEntityData.microserviceName && !otherEntityData.clientRootFolder) {
-                        otherEntityData.clientRootFolder = otherEntityData.microserviceName;
+                    if (otherEntityData) {
+                        if (otherEntityData.microserviceName && !otherEntityData.clientRootFolder) {
+                            otherEntityData.clientRootFolder = otherEntityData.microserviceName;
+                        }
+                        relationship.otherEntityIsEmbedded = otherEntityData.embedded;
```

Comment:
```
if (otherEntityData.embedded) {
    relationship.otherEntityIsEmbedded = true;
}
```
Otherwise otherEntityIsEmbedded will be written to entity.json for every relationship.